src/components/List/list.js

```diff
@@ -0,0 +1,106 @@
+import { useState } from "react";
+import "./list.css";
+import TodoAddTaskInput from "../input/input";
+import "bootstrap/dist/css/bootstrap.min.css";
+import * as Icon from "react-bootstrap-icons";
+
+const TodoList = () => {
+  const [list, setList] = useState([
+    { name: "one", complete: false, edit: false },
+    { name: "two", complete: false, edit: false },
+    { name: "three", complete: false, edit: false },
+  ]);
+  const [todoTaskForEditing, setTodoTaskForEditing] = useState("");
+  const [editingInProcess, setEditingInProcess] = useState(false);
+  const showTaskCompleted = (complete, index) => (event) => {
+    const { checked } = event.target;
+    const newArr = list.map((item, i) => {
+      if (checked === true) {
```

Comment:
improve this if else statement, you written duplicate code for a single check. Do it simply. @PuniaMohit 


if (checked === true) {
        if (index === i) {
          return { ...item, [complete]: true };
        } else {
          return item;
        }
      } else {
        if (index === i) {
          return { ...item, [complete]: false };
        } else {
          return item;
        }
      }